### DISCUSSION TITLE
docs/v1.12.x/en/reference/underlay-topology/

### DISCUSSION THREAD

**Discussion by giscus** [kubeovn/kube-ovn#3604] [Announcements]
# docs/v1.12.x/en/reference/underlay-topology/

https://kubeovn.github.io/docs/v1.12.x/en/reference/underlay-topology/

**Comment by WindHenry** [comment]
Hi everyone, I have some questions. How can we watch the config of physical switches? Can we config them? Thank you so much!!!

**Reply by WindHenry** [reply]
And Can we add more switches to create a topology in k8s ???
